client/extensions/archive-search/src/widget.tsx

```diff
@@ -0,0 +1,189 @@
+import moment from "moment";
+import * as React from "react";
+import { ISearchPanelWidgetProps, IVocabularyItem } from "superdesk-api";
+import {
+  Input,
+  MultiSelect as SuperdeskMultiSelect,
+} from "superdesk-ui-framework/react";
+import { superdesk } from "./superdesk";
+
+const { gettext } = superdesk.localization;
+
+interface IParams {
+  from: string;
+  to: string;
+  slugline: string;
+  headline: string;
+  story_text: string;
+  byline: string;
+  distribution: string[];
+  categories: string[];
+  languages: string[];
+  source: string[];
+}
+
+type MultiSelectProps = Record<
+  string,
+  {
+    label: string;
+    onChange: (
+      setParams: (params: Partial<IParams>) => void
+    ) => (selected: IVocabularyItem[]) => void;
+  }
+>;
+
+const multiSelects: MultiSelectProps = {
+  distribution: {
+    label: gettext("Services"),
+    onChange: (setParams) => (selected) => {
+      setParams({ distribution: selected.map((s) => s.qcode) });
+    },
+  },
+  categories: {
+    label: gettext("Wire"),
+    onChange: (setParams) => (selected) => {
+      setParams({ categories: selected.map((s) => s.qcode) });
+    },
+  },
+  languages: {
+    label: gettext("Languages"),
+    onChange: (setParams) => (selected) => {
+      setParams({ languages: selected.map((s) => s.qcode) });
+    },
+  },
+  source: {
+    label: gettext("Info source"),
+    onChange: (setParams) => (selected) => {
+      setParams({ source: selected.map((s) => s.qcode) });
+    },
+  },
+};
+
+const MultiSelect = ({
+  vocabularyKey,
+  label,
+  value,
+  onChange,
+}: Omit<MultiSelectProps[string], "label" | "onChange" | "defaultValue"> & {
```

Comment:
`Omit<MultiSelectProps[string], "label" | "onChange" | "defaultValue">`

Is there a purpose to use this type if everything is omitted? `defaultValue` property doesn't exist at all to be omitted.